projects/plugins/jetpack/_inc/client/scss/variables/_colors.scss

```diff
@@ -37,6 +37,9 @@ $gray-text-min: darken( $gray, 18% ); //#537994
 //
 // See wordpress.com/design-handbook/colors/ for more info.
 
+// Jetpack colors
+$jetpack-green-50: #008710;
```

Comment:
I think you could use `$studio-jetpack-green-50`, defined in `projects/plugins/jetpack/modules/calypsoify/_studio-wpcom.scss`.

Reply:
Looks like I didn't actually end up using this, I will remove it